resources/META-INF/plugin.xml

```diff
@@ -570,6 +570,9 @@
       <action id="flutter.clean" class="io.flutter.actions.FlutterCleanAction"
               text="Flutter Clean"
               description="Run 'flutter clean'"/>
+      <action id="flutter.makeHostAppEditable" class="io.flutter.actions.FlutterMakeHostAppEditableAction"
+              text="Flutter Make Host App Editable"
```

Comment:
I'm not sure whether we want this as `Flutter Make Host App Editable` - which makes other commands - or as `Make Flutter Host App Editable`, which is a bit more readable.